index.js

```diff
@@ -12,10 +14,18 @@ module.exports = (input, options) => {
   }
 
   options = Object.assign({
-    stringify: v => typeof v === 'undefined' ? '' : String(v)
-  }, options)
+    stringify: v => typeof v === 'undefined' ? '' : String(v),
+  }, options, {
+    wrap: Object.assign({
```

Comment:
These args should be reordered - the object containing the `wrap` options is currently overriding anything provided by the user rather than applying defaults.

We should also think about the CLI here. It'd be easier to handle these options at the CLI if they aren't in a nested object, so using `wrapWidth` and `wrapGutters` would need less tomfoolery.

Reply:
Yeah, I added a `gfm` (GitHub Flavored Markdown) option to quickly disable wrapping, replace newlines with breaks and pipes `|` with `\|`, so it doesn't really matter as we have to "spell out" the options anyway.